ag_data/formulas/library/system/mercury_formulas.py

```diff
@@ -0,0 +1,50 @@
+#
+# Formulas for Mercury's built-in and natively supported sensor types
+# Sample processing formulas with primary sensors and fictional Mercury-branded sensors
+#
+
+from ag_data import models
+
+
+def fPass(timestamp, sensor, measurement_payload):
+    result = {}
+
+    return result
+
+
+def fMercurySimpleTemperatureSensor(timestamp, sensor, measurement_payload):
+    result = {}
+
+    return result
+
+
+def fMercuryDualTemperatureSensor(timestamp, sensor, measurement_payload):
+    mean = measurement_payload["internal"] / 2 + measurement_payload["external"] / 2
+    diff = measurement_payload["internal"] - measurement_payload["external"]
+
+    result = {"mean": mean, "diff": diff}
```

Comment:
Nit: We can just return this dict literal without introducing a local variable `result`.

Reply:
Yes, we could. I think using a `result` variable makes formula functions more consistent. I need to check if this will create extra complexity, for example, if the python REPL will optimize it so the variable does not actually exist.

Reply:
I think no matter whether the python runtime would optimize it or not, time complexity here doesn't matter. It's all about conciseness.

Reply:
Good point. Changed this.

Reply:
This is now in #376.